src/main/java/com/autotune/common/annotations/json/Exclude.java

```diff
@@ -0,0 +1,11 @@
+package com.autotune.common.annotations.json;
```

Comment:
LICENSE header missing

Reply:
Thanks for the review @dinogun . Added the LICENSE Header